images/manageiq-base-worker/Dockerfile

```diff
@@ -7,6 +7,7 @@ COPY container-assets/ /vddk/
 
 RUN /vddk/extract-vmware-vddk
 
+################################################################################
```

Comment:
This makes it easier to see various multi-stage builds - I can remove from this PR if you want, but I figured it was fine.